Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "ed25519-bip32"
 version = "0.4.1"
-authors = ["Vincent Hanquez <vincent.hanquez@iohk.io>", "Nicolas Di Prima <nicolas.diprima@iohk.io>"]
+authors = ["Vincent Hanquez <vincent.hanquez@iohk.io>", "Nicolas Di Prima <nicolas.diprima@iohk.io>", "Ahmed Moussa <ahmed.moussa@iohk.io>"]
```

As we no longer modify the original lib but creating our own wrapper
```suggestion
authors = ["Vincent Hanquez <vincent.hanquez@iohk.io>", "Nicolas Di Prima <nicolas.diprima@iohk.io>"]
```